geode-core/src/main/java/org/apache/geode/management/GatewaySenderMXBean.java

```diff
@@ -185,6 +185,15 @@
       target = Target.GATEWAY)
   void start();
 
+  /**
+   * Starts this GatewaySender and cleans previous queue content.
+   * Once the GatewaySender is running its configuration cannot be changed.
+   *
+   */
+  @ResourceOperation(resource = Resource.CLUSTER, operation = Operation.MANAGE,
+      target = Target.GATEWAY)
+  void startWithClean();
```

Comment:
Can you change the method name to something more meaningful?, `startWithCleanQueue`, `startWithEmptyQueue`, something similar?

Reply:
Updated